schema.graphql

```diff
@@ -327,6 +327,7 @@ type FileMetadata {
 input FileMetadataFilters {
   closureMetadata: Boolean = false
   descriptiveMetadata: Boolean = false
+  additionalProperties: [String!]
```

Comment:
I would rename this to simply `properties` as its just a filter of the property names.